colour/io/tests/test_tm2714.py

```diff
@@ -170,6 +171,50 @@ def test_required_attributes(self):
         for attribute in required_attributes:
             self.assertIn(attribute, dir(Header_IESTM2714))
 
+    def test__eq__(self):
+        """
+        Tests :meth:`colour.io.tm2714.Header_IESTM2714.__eq__ method.
```

Comment:
The backtick is missing at the end here ``:meth:`colour.io.tm2714.Header_IESTM2714.__eq__`` and on all the subsequent docstrings.